docs/HACKING.md

```diff
@@ -51,120 +49,106 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
 
-There are some convenience scripts in `$PROJECT_HOME/goldstone-server/bin` for starting and stopping the virtual machines and docker containers that support the goldstone test environment.  For developer flexibilty, starting/stopping the django application has been omitted from the scripts.  To start the development environment, execute:
+Execute the following commands to configure the development OpenStack instance to ship logs and events back to Goldstone.  This only needs to be performed once:
 
-    $ workon goldstone-server
-    $ cd $PROJECT_HOME/goldstone-server
+    $ cd ~/devel/goldstone-server
     $ ./bin/start_dev_env.sh
 
-Then, in a new window, execute:
-    $ workon goldstone-server
-    $ cd $PROJECT_HOME/goldstone-server
-    $ fab runserver   # select `local_docker` settings
-
-To stop the development environment:
+Then in another window:
 
-In the window running `fab runserver`:
-   
-    $ <Ctrl-C>  # exits the Django server
+    $ eval $(docker-machine env default)
+    $ ./bin/configure_stack.sh
 
-In the window running `start_dev_env.sh`:
 
-    $ <Ctrl-C>  # executes `stop_dev_env.sh`
+It may be helpful to create a couple of instances via the API in order to generate some log, event, and metric activity.  You can execute the following commands to create a small instance:
 
-or: 
+    $ eval $(docker-machine env default)
+    $ docker exec -i -t goldstoneserver_gsappdev_1 bash -i -c 'nova boot --image cirros --flavor m1.tiny ceilo0'
 
-    $ workon goldstone-server
-    $ cd $PROJECT_HOME/goldstone-server
-    $ ./bin/stop_dev_env.sh
+Here are some [screenshots](https://photos.google.com/album/AF1QipPsFIXlFUzuJflAowyshNoDtF3ph9hMAIdK4WGa) of a working dev environment. Your environment should look similar.
 
-## What to Expect When Running For the First Time
 
-The first time you start Golstone Server, you may see errors and missing data in the user interface. You may aloso see failures if you execute the test suite.  The data should be sufficiently populated in 10 minutes.  If you continue to see errors in the UI or in tests, please submit an issue!
+## Logging In
 
-It may be helpful to create a couple of instances via the API in order to generate some log, event, and metric activity.  You can execute the following commands to create a small instance:
+After the containers have started and OpenStack has been configured to interact with Goldstone, you can access the user interface with the following information:
 
-    $ workon goldstone-server
-    $ cd $PROJECT_HOME/goldstone-server
-    $ nova boot --image cirros --flavor m1.tiny ceilo0
+* url: **http://127.0.0.1:8000**
+* django admin username: **admin**
+* django admin password: **goldstone**
+* Goldstone admin username: **gsadmin**
+* Goldstone admin password: **goldstone**
 
-Here are some [screenshots](https://photos.google.com/album/AF1QipPsFIXlFUzuJflAowyshNoDtF3ph9hMAIdK4WGa) of a working dev environment. Your environment should look similar.
 
+## Testing
```

Comment:
Somewhere, probably here before the Testing section, you need to add sections on where to find the celery task output, and how to get a Python prompt and/or shell prompt in the container.